harvard-lts/drs-translation-service
# ETD Load Report Service

**Adds the load report service for ETDs and writes a message to the ETD Alma DRS Holding Queue upon retrieval*
* * *

**JIRA Issue**: https://at-harvard.atlassian.net/browse/ETD-336

# How should this be tested?

1. Retrieve the latest .env for DTS from Dee Dee
2. Follow the [readme](https://github.com/harvard-lts/drs-translation-service/blob/main/README.md#testing) to run the tests
3. Exec into the local docker container and run `python3 scripts/invoke-task.py`
4. Open Jaeger dev and view the latest ETD traces.  The "ETD: ALMA DRS HOLDINGS - add_holdings" trace should be visible with the hello world default being called.  You should see the identifier in the trace.

Steps 3 and 4 do the following: 1. DTS publishes a message on the etd_ingested_into_drs queue. 2. ETD Alma DRS Holding Service consumes the message and executes.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? yes
- integration tests? no
